plugin-enclave-gradle/src/main/kotlin/com/r3/conclave/plugin/enclave/gradle/LinuxExec.kt

```diff
@@ -118,6 +118,7 @@ open class LinuxExec @Inject constructor(objects: ObjectFactory) : ConclaveTask(
             "run",
             "-i",
             "--rm",
+            "-u", "\$(id -u):\$(id -g)",
```

Comment:
I don't think this will work here. I believe `$(...)` is bash syntax only and is a shortcut to execute a command. You will probably need to call `id` as a separate process exec.

Reply:
Solved this.